assets/js/vue/components/information-product/InformationProductForm.vue

```diff
@@ -370,8 +370,8 @@ export default {
         ).then(() => {
           this.ipSuccessModalText = 'Information Product is Updated!';
           this.ipCreatedSuccessModal = true;
-        }).catch(() => {
-          this.errorMessage = 'Unable to update Information Product';
+        }).catch((response) => {
```

Comment:
Maybe this should be name error, because that's what is being return, a struct based on class Error. Which by itself also has a response attribute in it.